qa/tasks/deepsea.py

```diff
@@ -124,6 +125,46 @@ def __init__(self, ctx, config):
         # self.log.debug("ctx.config {}".format(ctx.config))
         # self.log.debug("deepsea context: {}".format(deepsea_ctx))
 
+    def _deepsea_minions(self):
+        """
+        Set deepsea_minions pillar value
+        """
+        echo_cmd = (
+            'echo "deepsea_minions: \'*\'" > '
+            '/srv/pillar/ceph/deepsea_minions.sls'
+        )
+        self.master_remote.run(args=[
+            'sudo',
+            'sh',
+            '-c',
+            echo_cmd,
+            run.Raw(';'),
+            'cat',
+            '/srv/pillar/ceph/deepsea_minions.sls',
+            ])
+
+    def _deepsea_version(self):
+        if self.deepsea_cli:
+            try:
+                self.master_remote.run(args=[
+                    'type',
+                    'deepsea',
+                    run.Raw('>'),
+                    '/dev/null',
+                    run.Raw('2>&1'),
+                    ])
+            except CommandFailedError:
+                raise ConfigError(
+                    "(deepsea task) test case calls for deepsea CLI, "
+                    "but it is not installed"
+                    )
+            self.master_remote.run(args='deepsea --version')
```

Comment:
Why not run this in try block?

Reply:
because the try block has an error message specific to its contents

Reply:
Looks like you didn't understand my comment, you do not need to call type, because running deepsea --version will fail only if it is not installed and you also can report `but it is not installed` in this case, but if it is installed it will only print output to stdout.

Reply:
Yeah, there's a misunderstanding here. I only want to run `deepsea --version` if the command is available, that's why I'm running type first. Let's go on to the next topic.